### PR TITLE
Factor data frame

### PR DESCRIPTION
**Pull request by Lunalo**
This is open for discussion if instead i can proceed to edit the method.

### CODE REVIEW

**Comment by Lunalo** [comment]
@dannyparsons . I need your opinion on this. I am happy proceeding to edit R method

**Comment by dannyparsons** [comment]
Yes happy for you to edit the R method for this.

File: instat/dlgFactorDataFrame.vb [hunk 1]
```diff
@@ -47,6 +47,7 @@ Public Class dlgFactorDataFrame
     End Sub
 
     Private Sub ReopenDialog()
+        ucrSelectorFactorDataFrame.Reset()
     End Sub
```
Comment (maxwellfundi) [inline instat/dlgFactorDataFrame.vb]: Why are you reseting the selector on the reopen? Do you want to clear receivers everytime the dialog reopens?

**Comment by maxwellfundi** [comment]
@Lunalo was the method edited? Is this ready?

**Pull request merged**